client/pages/Info/Info.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import './Info.scss';
+
+
+const Info = () => (
+  <div className="main-info">
+    <div className="Map">
+      <p>Here will be map</p>
+    </div>
+    <div className="contacts">
+      <h1>Contacts</h1>
+      <p>Our phone 66-666-666</p>
+      <p>Our email ad@666.gmail.com</p>
+    </div>
+    <div className="form">
+      <form>
+        <input type="text" name="name" placeholder="input your name\login" required /> <br />
+        <input type="email" name="email" placeholder="input your email" required /> <br />
+        <textarea required placeholder="your message here">
+          Your message
+        </textarea>
```

Comment:
Use the `defaultValue` or `value` props instead of setting children on <textarea>